### PR TITLE
fix: 1740 max performance milestone calculation

### PR DESCRIPTION
**Pull request by BCerki**
card: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1740

### CODE REVIEW

File: app/cypress/e2e/cif/project-revision/create-first-project-revision.cy.js [hunk 1]
```diff
@@ -112,6 +113,7 @@ describe("when creating a project, the project page", () => {
     );
     cy.get('[aria-label="Holdback Amount This Milestone"]').contains("$12.00");
     cy.happoAndAxe("Project milestone reports Form", "filled", "main");
+    cy.pause();
```
Comment (BCerki) [inline app/cypress/e2e/cif/project-revision/create-first-project-revision.cy.js:116]:
```suggestion
```

**Review by Sepehr-Sobhani — APPROVED** [review]
🥳

File: schema/test/unit/computed_columns/form_change_maximum_performance_milestone_amount_test.sql [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+),
+(
+  4,
+  'create',
+  'cif',
+  'reporting_requirement',
+  3,
+  1,
+  'milestone',
+  '{"reportType": "Performance Milestone", "hasExpenses": false, "reportDueDate": "2022-11-10 15:09:36.264005-08", "adjustedHoldbackAmount": 20000, "reportingRequirementIndex": 4, "certifierProfessionalDesignation": "Professional Engineer"}'
```
[the remark this answers is not in the document]
Comment (BCerki) [reply schema/test/unit/computed_columns/form_change_maximum_performance_milestone_amount_test.sql]: The function checks for true/false, so a couple of them need to be false to test that they're correctly being left out

[the remark this answers is not in the document]
Comment (Sepehr-Sobhani) [reply schema/test/unit/computed_columns/form_change_maximum_performance_milestone_amount_test.sql:61]: That makes sense, sorry for the confusion!

**Pull request merged**